common/service/sensor/sensor.c

```diff
@@ -271,22 +271,22 @@ __weak void pal_fix_sensor_config(void)
 	return;
 }
 
-bool stby_access(uint8_t sensor_number)
```

Comment:
Common code may conflict with upper source(OpenBIC), should skip this error until next merge.

Reply:
This is cherry-picked from the [PR#9](https://github.com/MouchenHung-QUANTA/OpenBIC/pull/9) to pass the OpenBIC Build and Analyze processes. I will remove the commit from this PR if you want, but the build and analyze will fail.